synapse/lib/config.py

```diff
@@ -129,7 +129,7 @@ def getConfDefs(self):
         '''
         Returns the configuration definitions for this object.
         '''
-        return {name: dict(info) for (name, info) in self._conf_defs.items()}
+        return {name: dict(info[1]) for (name, info) in self._conf_defs.items()}
```

Comment:
why was this changed from `dict(info)` to `dict(info[1])`?

Reply:
makes sense

Reply:
The info data is a tufo form.